packages/blockchain-wallet-v4-frontend/src/components/Flyout/model.tsx

```diff
@@ -866,15 +866,19 @@ type BankProps = {
   bankDetails: BankDetails
   icon: ReactElement
   isActive: boolean
+  isDisabled?: boolean
   onClick: () => void
   text: string
 }
 
-const Bank = ({ bankDetails, icon, isActive, onClick, text }: BankProps) => (
+const Bank = ({ bankDetails, icon, isActive, isDisabled = false, onClick, text }: BankProps) => (
   <DisplayContainer
     data-e2e={`sb${bankDetails?.bankAccountType?.toLowerCase()}Banks`}
     role='button'
-    onClick={onClick}
+    onClick={!isDisabled ? onClick : undefined}
+    style={{
+      opacity: isDisabled ? '0.5' : '1'
+    }}
```

Comment:
Can we instead introduce `isDisabled` prop for `DisplayContainer` which would add this style?